src/components/ProgramCollection.js

```diff
@@ -108,151 +134,227 @@ class ProgramCollection extends Component {
       defaultMessage: 'Sort',
     });
 
+    const Title = withStyles(() => ({
+      root: {
+        fontWeight: 'bold',
+      },
+    }))(Typography);
+
+    const NonCollapsingContainer = withStyles((theme) => ({
+      root: {
+        [theme.breakpoints.up('xs')]: {
```

Comment:
The `Container` component does great at setting the `maxWidth`. But, I needed a `minWidth`, too, to keep everything from collapsing when a search/filter resulting in few or zero programs.
This  solution doesn't look great, but it seems to be working really well.